js/iframeResizer.contentWindow.js

```diff
@@ -1001,7 +1007,7 @@
 					setTimeout(function(){ initLock = false;},eventCancelTimer);
 				}
 
-				if (document.body){
+				if (document.readyState === "interactive" || document.readyState === "complete"){
```

Comment:
This line has been generated after grunt build. Probably part of previous commit.

Reply:
Thanks for this, could you please remove the `js` folder from the PR, I just need `src`.